tests/e2escenarios/e2e_scenarios_suite_test.go

```diff
@@ -1,13 +1,27 @@
 package e2escenarios
 
 import (
+	"fmt"
+	"os"
 	"testing"
+	"time"
 
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
 	. "github.com/onsi/gomega"
 )
 
 func TestE2eScenarios(t *testing.T) {
 	RegisterFailHandler(Fail)
-	RunSpecs(t, "odo e2e scenarios")
+	time := time.Now()
+	xmlFileName := fmt.Sprintf("../../reports/junit_%d-%d-%d_%02d-%02d-%02d.xml", time.Year(), time.Month(),
```

Comment:
What if two test scenarios are executed in the same second?

Reply:
Do we really care this ? It's hard to find such scenario in our current context because the spec are parallel but not make targets.

Reply:
But there are make targets that run multiple specs like https://github.com/openshift/odo/blob/01dc6245cadae9c817839e8377b1b6afa2430529/Makefile#L202

Reply:
The reporter file is generated in suite level. So multiple report file will be created only if you run multiple targets, not multiple spec in one target.